manuals/1.0/ja/12-philosophy-behind.md

```
@@ -7,406 +7,400 @@ permalink: /manuals/1.0/ja/12-philosophy-behind.html
 
-### なぜ存在論なのか
+古代哲学と現代のコードを結びつけるのは、格好つけるためではありません。これらを理解することで、馴染みのある問題を違う角度から見られるようになり、パターンがより直感的に感じられるかもしれない——そう思って紹介しています。
 
-従来のプログラミングは二つの問いに答えてきました：
+---
+
+## 1.「Tell」から「Be」へ
+
+### 強調点の違い
+
+**1967年：Tell, Don't Ask**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)